src/setup/util/odometry.cpp

```diff
@@ -0,0 +1,87 @@
+#include "setup/util/odometry.h"
+
+#include "setup/control/base.h"
+
+#define CONVERSION_FACTOR 0.000239982772149 //Centidegrees to inches
+
+//Distance between tracking wheel and center
+double leftOffset = 3.75; //3.75 in
+double rightOffset = 3.75;
+double backOffset = 6.75; //6.75 in
+
+//Initialization
+double leftPrev = 0;
+double rightPrev = 0;
+double backPrev = 0;
+
+double thetaPrev = 0;
+
+//Set start position and angle
+double x = 0;
+double y = 0;
+double heading = 0;
```

Comment:
I already see heading declared globally. Can we use a different name for this local variable